src/App.tsx

```diff
@@ -27,13 +27,13 @@ function App() {
 
   return (
     <PanelGroup autoSaveId="refstudio" direction="horizontal">
-      <Panel defaultSize={20} collapsible className="overflow-scroll p-4">
+      <Panel className="overflow-scroll p-4" collapsible defaultSize={20}>
```

Comment:
This change comes from `react/jsx-sort-props`.

Reply:
If this is auto-sortable on save it's ok for me.